src/main.cpp

```diff
@@ -199,6 +199,9 @@ void vcMain_LoadSettings(vcState *pProgramState, bool forceDefaults)
     case 1: ImGui::StyleColorsDark(); break;
     case 2: ImGui::StyleColorsLight(); break;
     }
+
+    SDL_SetWindowSize(pProgramState->pWindow, pProgramState->settings.window.width, pProgramState->settings.window.height);
```

Comment:
There's a good chance that this has broken the web client, I don't recall the exact rules when it came to the `SDL_SetWindowSize` function but there's a situation that will cause it to not resize with the browser window.